rasa/core/tracker_store.py

```diff
@@ -198,11 +199,12 @@ def _deserialise_dialogue_from_pickle(
         sender_id: Text, serialised_tracker: bytes
     ) -> Dialogue:
 
-        logger.warning(
-            f"DEPRECATION warning: Found pickled tracker for "
+        warnings.warn(
```

Comment:
that's also a `logger.warning`. User can't do anything. The tracker stores handles this itself.